bundle/language_en.properties

```diff
@@ -882,6 +885,7 @@ angal.medicalstock.refno.col
 angal.medicalstock.selectionpanel                                                                      = Selection Panel
 angal.medicalstock.stockmovement.title                                                                 = Stock Movement
 angal.medicalstock.stockmovementbrowser.title                                                          = Stock Movement Browser
+angal.medicalstock.deletemovement.msg                                                                  = Do you really want to delete the last Movement?
```

Comment:
In the two other messages you added `movement` is lowercase; here it is upper.   This seems inconsistent.